src/operator/tensor/elemwise_binary_op_extended.cc

```diff
@@ -26,9 +26,8 @@
 
 namespace mxnet {
 namespace op {
-MXNET_OPERATOR_REGISTER_BINARY(_power)
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(_power, mshadow_op::power)
```

Comment:
I think power should be handled differently: 
if input is rsp/csr, and scalar is not 0, then the output is sparse. 
In inferStorageType, you're able to read the value for the scalar from the param. This will make this operator more efficient.